spec/sources/buildkite_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'danger/ci_source/circle'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.